docs/conf.py

```diff
@@ -365,5 +368,10 @@
 # Add configuration for intersphinx mapping
 
 intersphinx_mapping = {
-    'starter-pack': ('https://canonical-example-product-documentation.readthedocs-hosted.com/en/latest', None)
+    'starter-pack': ('https://canonical-example-product-documentation.readthedocs-hosted.com/en/latest', None),
+    'sphinxcontrib-mermaid': ('https://sphinxcontrib-mermaid-demo.readthedocs.io/en/latest', None)
 }
+
+# Mermaid configuration
+
+mermaid_d3_zoom = True
```

Comment:
I strongly suggest we not enable this by default. It captures mouse input while scrolling down the page, which is a UX issue. Embedded Google Maps frames used to have this problem, which is why added the `Ctrl` modifier to their interface as a workaround.

I think by leaving it off by default, it encourages better design and thinking about diagrams in our contributors – we want them to find ways to make sensible, legible, diagrams that fit on the page width that we've provided for them.

If people have truly complex diagrams, we could point them to the `:zoom:` option in the directive.